### DISCUSSION TITLE
Disable `Forgot Password`

### DISCUSSION THREAD

**Discussion by q-wertz** [requarks/wiki#5822] [Help / Questions] [answered]
Is it possible to disable the `Forgot Password` function?

Did not find a related setting. It would be useful on a per authentication method basis. For e.g. Active Directory one could completely disable the link or link it to a webpage where the AD password can be resetted/contact to the AD administrators is given.

**Comment by NGPixel** [comment, marked as answer]
You can hide the local authentication provider under Admin > Security, which will hide the forgot password link as well.

**Comment by q-wertz** [comment]
1. Tried to disable it, but for the AD authentication method the link is still shown. It then asks for the email and tries to send a email which doesn't make any sense, because the WikiJS cannot change the AD password
 
    ![Screenshot from 2022-11-09 10-56-52](https://user-images.githubusercontent.com/20948931/200800422-ead5df26-e543-4f3f-9081-557987acfbc5.png)


2. It would be nice to be able to configure that on a per authentication method basis (e.g. local is handled by WikiJS directly, Active Directory by some specific website or disabled at all, …)

**Comment by jikaven** [comment]
Hello!
This is still a thing for LDAP connections nearly 2 years later.
Even using `?all` and accessing the local login tab, the button still appears.
Any chance this might get looked upon once more?
